### ISSUE TITLE
Some problems with bat file on Windows

### ISSUE THREAD

**Issue by moteus** [luarocks/luarocks#930]
I just try install busted and found couple problems.
1. bat file has mixed EOL (some lines have unix style and some - unix).
(all commands like `@echo off` ends with `CR` and all empty line has `CRLF`)
Windows does not recognize `CR` as EOL.
2. bat file did not copied to to `tree/bin` directory placed to `\systree\lib\luarocks\rocks-5.1\busted\2.0.rc13-0\bin\busted.bat`

Also I have couple suggestion about bat file
1. Please add `setlocal` to bat wrapper bat file. So running such bat file did not affect to env of command shell.
2. May be add ability to add some prefix to bat file. (e.g. if I have 3 Lua version on my system and I want 3 version of busted). But it can lead to some other problems.

**Comment by daurnimator** [comment]
> May be add ability to add some prefix to bat file. (e.g. if I have 3 Lua version on my system and I want 3 version of busted). But it can lead to some other problems.

busted accepts `--lua` to pick a lua executable.

**Comment by moteus** [comment]
> busted accepts --lua to pick a lua executable.

Yes. But bat wrapper redefine LUA_PATH/LUA_CPATH so as resulte there may be some problem
